doc/guides/zephyr_cmake_package.rst

```diff
@@ -0,0 +1,231 @@
+.. _cmake_pkg:
+
+Zephyr CMake Package
+####################
+
+Zephyr can be exported as a CMake config package.
+
+A CMake config package allows a developed to integrate Zephyr into their own application by using
+``find_package(Zephyr)``.
+
+Exporting Zephyr as a CMake package is identical to create a reference to the current Zephyr
+installation inside the CMake user package registry.
+
+
+.. tabs::
+
+   .. group-tab:: Ubuntu
+
+      In Linux, the CMake user package registry is found in:
+
+      ``~/.cmake/package/Zephyr``
+
+   .. group-tab:: macOS
+
+      In macOS, the CMake user package registry is found in:
+
+      ``~/.cmake/package/Zephyr``
+
+   .. group-tab:: Windows
+
+      In Windows, the CMake user package registry is found in:
+
+      ``HKEY_CURRENT_USER\Software\Kitware\CMake\Packages\Zephyr``
+
+
+The Zephyr package allows CMake to automatically find a Zephyr base.
+At minimum, a single Zephyr installation must be exported, but it is allowed to export multiple.
+Exporting multiple Zephyr installations may be useful when developing applications out-of-tree.
```

Comment:
```suggestion
Exporting multiple Zephyr installations may be useful when developing applications out-of-tree in their own workspaces, using vendor forks, etc.
```

Reply:
Updated, but might need a second update, depending on `out-of-tree` naming decision.

Reply:
done.